web/testing/cypress/e2e/ID009_Remove_Event_From_Schedule.cy.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable no-undef */
+/// <reference types="cypress" />
+
+describe("Remove Event From Schedule", () => {
+    beforeEach(() => {
+      cy.viewport(1440, 900);
+  
+      cy.visit("/");
+    });
+  
+  it("Remove Event From Schedule", () => {
+    //log in
+    const userEmail = `e2e+${Math.ceil(
+        Math.random() * 1000000000000000
+      )}@testdfas.com`;
+      console.log(userEmail);
+      cy.get(".w-full > .flex").click();
+      cy.origin(
+        "https://oncampus.us.auth0.com",
+        { args: { userEmail } },
+        ({ userEmail }) => {
+          cy.get("a").contains("Sign up").click();
+          cy.get("input#email").type(userEmail); //me might have to go through signup every time and create a new account? might trigger the auth0 limits if we do that...
+          cy.get("input#password").type("#e5PUJe9@mroL$", { log: false });
+          cy.contains("button[value=default]", "Continue").click();
+          cy.get("button").contains("Accept").click();
+        }
+    );
+  
+    //create volunteer event
+    cy.visit("/schedule");
+    cy.wait(5000);
+    cy.get(":nth-child(5) > :nth-child(2)").dblclick();
+    cy.get("input[placeholder=Title]").type("Volunteer", { force: true }).should('have.value', 'Volunteer');
+    cy.get('input[placeholder="Add description"]').type("D", { force: true });
+    cy.get('input[value="06/02/2023 08:00 AM"]').click().clear().type("06/02/2023 09:00 AM", {delay: 100}, { force: true }); //edit time 
```

Comment:
Consider getting the input field with something like this instead, so that it continues working even if /schedule leads to the current date's week and not the week of Feb 5th 2023: 

cy.get('input[placeholder="dd/mm/yyyy hh:mm (a|p)m"]').eq(0).clear();
cy.get('input[placeholder="dd/mm/yyyy hh:mm (a|p)m"]').eq(0).should("be.enabled").type("06/02/2023 09:00 AM");
cy.get('input[placeholder="dd/mm/yyyy hh:mm (a|p)m"]').eq(1).clear();
cy.get('input[placeholder="dd/mm/yyyy hh:mm (a|p)m"]').eq(1).should("be.enabled").type("06/02/2023 06:00 PM");